Update code comment for mongoose sample query (#29865)
mdn/content: files/en-us/learn/server-side/express_nodejs/mongoose/index.md

const Athlete = mongoose.model("Athlete", yourSchema);

> // find all athletes who play tennis, selecting the 'name' and 'age' fields
> // In other words, select the 'name' and 'age' fields for all athletes who play tennis

Reviewer: Thank you. I am not sure that this paraphrase adds much you can't infer from the original text. However you wouldn't have added the correction if you didn't think the original was confusing, so I have modified the text to make it crystal clear (essentially people don't need to know that SELECT is the SQL term for this behaviour).

```suggestion
// Find all athletes who play tennis, returning the 'name' and 'age' fields
```
